design-proposals/primary_network_binding_for_seamless_migration.md

```diff
@@ -0,0 +1,520 @@
+| IP & Route delivery         | Self (IPv4 & IPV6)                                                                                              | Self for IPv4, Net Provider for IPv6 | Net Provider                                     |
+| Pod Probes Support          | Yes                                                                                                             | No                                   | No                                               |
+| Seamless Live-Migration     | Requires Development                                                                                            | Yes                                  | Yes                                              |
+| Supported Traffic           | TCP/UDP/ICMP (All IP in case seamless migration is resolved)                                                    | IP                                   | IP                                               |
+| Migration Support Indicator | Yes                                                                                                             | Requires Development                 | Yes                                              |
+| Mesh / Istio support        | Yes                                                                                                             | No                                   | No                                               |
+| IP Status (reporting)       | Yes                                                                                                             | Yes                                  | Requires Development                             |
+| Observability Integration   | Yes                                                                                                             | No                                   | No                                               |
+| Resource Overhead           | 500MB per VM, 250MB per VM already available on U/S, Negligible overhead when resolving the seamless migration. | Negligible                           | Negligible                                       |
+| Performance                 | N/A                                                                                                             | N/A                                  | N/A , Most likely better than the bridge option. |
+| Kubevirt Support            | Low                                                                                                             | High                                 | Medium                                           |
+
+[*] Core Bridge binding can be deployed as a binding plugin with a subset
+    of its existing feature sets. E.g. as a plugin, IP & routes can be
+    supported only through the network-provider.
+
+### The Binding Winner
```

Comment:
Could you please update this section following the results of the macvtap CNI PoC?

Reply:
I also updated the results in the specific sections.

Done